### PR TITLE
ADAPT-8912 - react version of text input

### PR DESCRIPTION
**Pull request by eleanor-heath**
https://github.com/adaptlearning/adapt-contrib-textInput/issues/110

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by lc-alexanderbenesch — APPROVED** [review]
👍

**Review by joe-allen-89 — APPROVED** [review]
👍

File: templates/textinput.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  const {
+    isInteractive,
+    _canShowMarking,
+    _isInteractionComplete,
+    _id,
+    _isEnabled,
+    _isCorrect
+  } = props;
+
+  const shouldShowMarking = isInteractive() && _canShowMarking;
```
Comment (oliverfoster) [inline templates/textinput.jsx:14]: Can use `_shouldShowMarking` straight from the model

**Review by oliverfoster — CHANGES_REQUESTED** [review]

**Comment by oliverfoster** [comment]
Just waiting for https://github.com/adaptlearning/adapt-contrib-core/pull/68 to be merged and released.

**Pull request merged**